docs/api.md

```diff
@@ -28,7 +28,7 @@ mainly, but not only, provisioning details.
     * Redfish
       * `redfish://` (or `redfish+http://` to disable TLS), the hostname
         or IP address, and the path to the system ID are required,
-        for example `redfish://myhost.example/redfish/v1/Systems/MySystemExample`
+        for example `redfish://myhost.example/redfish/v1/Systems/1`
```

Comment:
The real world example we have is `/redfish/v1/Systems/System.Embedded.1`, would that be better?

Reply:
I was just thinking we might want to indicate it doesn't need to be numeric

However, https://docs.openstack.org/ironic/latest/admin/drivers/redfish.html shows your example here, so maybe it's a better idea to match the Ironic docs.

Reply:
I've updated the patch to replace the existing value with a real one and to add the example from the ironic documentation.

Reply:
:+1: